docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/LightGBMBinaryClassification.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using Microsoft.ML.Transforms.Categorical;
 
-namespace Microsoft.ML.Samples.Dynamic
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
 {
-    public class LightGbmBinaryClassification
+    public class LightGbm
```

Comment:
>LightGbm [](start = 17, length = 8)

Please also rename the file to LightGbm.cs

Reply:
@shmoradims Done